akka-distributed-data/src/test/scala/akka/cluster/ddata/FlagSpec.scala

```diff
@@ -32,13 +32,15 @@ class FlagSpec extends WordSpec with Matchers {
 
     "have unapply extractor" in {
       val f1 = Flag.Disabled.switchOn
-      val Flag(value1) = f1
-      val value2: Boolean = value1
+      val Flag(_: Boolean) = f1
```

Comment:
I think this was exercising usage, better to add some `value1 should ===...` to use the values